stl/inc/xtzdb.h

```diff
@@ -78,14 +78,46 @@ void __stdcall __std_tzdb_delete_sys_info(__std_tzdb_sys_info* _Info) noexcept;
 
 __std_tzdb_registry_leap_info* __stdcall __std_tzdb_get_reg_leap_seconds(
     size_t _Prev_reg_ls_size, size_t* _Current_reg_ls_size) noexcept;
-
 void __stdcall __std_tzdb_delete_reg_leap_seconds(__std_tzdb_registry_leap_info* _Rlsi) noexcept;
 
 _NODISCARD void* __stdcall __std_calloc_crt(size_t _Count, size_t _Size) noexcept;
 void __stdcall __std_free_crt(void* _Ptr) noexcept;
 
 _END_EXTERN_C
 
+template <class _Ty>
+struct _Tzdb_deleter {
+    void operator()(_Ty*) const noexcept {}
+};

```

Comment:
I think we should declare the primary template but not provide a class definition - that way, if we ever accidentally instantiate it for another type, we'll get a compiler error instead of a deleter that silently leaks allocations.